### PR TITLE
Merge dev into main

### PR DESCRIPTION
**Pull request by alexiscyber14**
Requirements:
-Work as a team(we were two)
- student A Alex ssenyond
- student B Esther
- Use Api to fetch data and post data
-  video link  https://www.loom.com/share/97ef3fb2bec3426187eaee52c35592cc

### CODE REVIEW

**Review by levy002 — COMMENTED** [review]
## Hello @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: 

You have submitted a project, that doesn't have a README. The `Dev` branch you submitted, doesn't have a README file. KIndli fi that and submit your project again for review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/modules/like.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ const liking = () => {
           count + 1;
           paragraph.textContent = count;
           sendLikesTotal(itemId, count);
+          window.location.reload();
         });
       });
     });
```
Comment (Qoosim) [inline src/modules/like.js:53]: - [ ] Wonderful job so far. However, there's little thing you need to work on before your project could be approved. 
When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated. I can see this feature was implemented but I noticed the interface does not update at first click. It takes a user to click more thaan one before the screen get updated.

[javascript-behave.webm](https://user-images.githubusercontent.com/26597432/226103584-d0172b55-637f-405f-812e-f6dc638c3ba7.webm)

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/like.js (hunk 1 above)
Comment (alexiscyber14) [reply src/modules/like.js:53]: Thank you. lemme  fix this this

Comment (alexiscyber14) [reply src/modules/like.js]: @Qoosim I have fixed it brother and sent the review. thank you.

**Review by sumairq — APPROVED** [review]
### Hi Team, 



In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**